Documentation/Signoffs/Power_System_Camera.md

```diff
@@ -1,37 +1,125 @@
+|  3 | The system shall be able to supply enough power to the camera system for its full functionality | Design Team | 
+|  4 | The system shall not be impacted by back EMF from the servo motors | Design Team | 
+|  5 | The system shall contain a manual switch to individually cut off power supplied | Standards and Safety | 
 
-<sup>1</sup> Reason for contraint 1 and why it is important
+Table 1 : Constraints <br />
 
-<sup>2</sup> Reason for contraint 2 and why it is important
+      
+<sup>1</sup> ASTM B258-18 is a standard for the proper wire gauge for electrical conductors. It is important to adhere to this standard to ensure the system is safe and avoid overheating, short circuits, and fires.  <br />
 
-<sup>3</sup> Reason for contraint 3 and why it is important
+<sup>2</sup> The system will be located outdoors, so it is essential that the electrical components will be protected from environemental weather hazards and damage. The IP67 rating protects the system from cover rain, snow, dust, wind, and UV. Using this NEMA box will improve the longevity and prevent damage of the system. <br />
+
+<sup>3</sup> This constraint is most important for system functionality, because the power system must supply the rated power for the camera system to acheive full functionality without running into any under voltage scenarios that could impact system peformance. <br />
+
+<sup>4</sup> This constraint ensures that the power supply will not be damaged by an induced back EMF from the servo motors. 
+<br />
+
+<sup>5</sup> This constraint ensures safety of the system by providing a "kill switch" to immediately remove the power supplied to both power supplies in an emergency scenario.
+<br />
 
 ## Schematic
 
+![Screenshot 2024-09-17 at 12 37 04 AM](https://github.com/user-attachments/assets/5a6c6241-79d9-4dab-8101-452bd22ecdc4)
+<br />
+Figure 1: Camera Power Buildable Schematic.<br />
+The above schematic shows the component connections for the system. We will be utilizing a wall outlet for this design then using a 15 ft two way extensions cord with switching functionality. One side of the extension powers the Raspberry Pi 5 V 15 W USB-C power supply while the other powers the DC 5 V 20 A 100 W power supply. The Raspberry Pi 5 V 15 W USB-C power supply provides power to the Raspberry Pi 4B then a CSI ribbon cable is used to provide power at 5 V from the Raspberry Pi 4B to the Raspberry Pi HQ camera. The DC 5 V 20 A 100 W power supply's outputs go through two current sensor modules to prevent damage to the supply from back EMF. From the modules, 5 V are supplied to both motors.
+<br />
+![Electrical_Overview](https://github.com/user-attachments/assets/0ce5b3c5-ff64-4a7f-b42c-370840f1fc91)
+<br />
+Figure 2: Electrical Overview.
+The above figure shows the simplified electrical module, providing 5 V to all components, and motor connections as detailed in the Camera Hardware system documentation. 
+
```

Comment:
Fig. 2 does not show current sensors. The data sheet shows the current sensors require power and have an output terminal, in addition to the input which is the current being sensed. How are these connected?

Reply:
Current sensors removed. Replaced with a diode to protect from EMF.